components/movementsensor/gpsnmea/gpsnmea.go

```diff
@@ -60,9 +60,7 @@ var model = resource.DefaultModelFamily.WithModel("gps-nmea")
 // NmeaMovementSensor implements a gps that sends nmea messages for movement data.
 type NmeaMovementSensor interface {
 	movementsensor.MovementSensor
-	Close(ctx context.Context) error                 // Close MovementSensor
-	ReadFix(ctx context.Context) (int, error)        // Returns the fix quality of the current MovementSensor measurements
-	ReadSatsInView(ctx context.Context) (int, error) // Returns the number of satellites in view
+	Close(ctx context.Context) error // Close MovementSensor
```

Comment:
Follow up pr - we do not need this interface at all. Close is a resource level fucntion.

Reply:
created https://viam.atlassian.net/browse/RSDK-8225